scripts/ci/install.sh

```diff
@@ -16,6 +16,7 @@ function retry {
 # work or not.
 retry pip install --upgrade pip setuptools wheel
 retry pip install .[all,tests]
+retry pip install git+https://github.com/spulec/moto
```

Comment:
Maybe latest tag?

Reply:
@pared There is none unfortunately. Will use git sha until guys release new version.